File: python/samples/people_picker_sample.py
from fluent_cards import AdaptiveCardBuilder


def create_people_picker_card() -> dict:
    """Creates a people picker card that searches users from Microsoft Graph."""
    return (
        AdaptiveCardBuilder.create()
        .with_version('1.6')
        .add_input_choice_set(
            lambda i: i.with_id('people-picker')
            .with_label('Select users in the whole organization')
            .is_multi_select()
            .with_value('user1,user2')
            .with_choices_data('graph.microsoft.com/users')
        )
        .add_action(lambda a: a.submit('Submit'))
        .build()
    )

File: node/samples/peoplePickerSample.ts
import { AdaptiveCardBuilder } from 'fluent-cards';

/** Creates a people picker card that searches users from Microsoft Graph. */
export function createPeoplePickerCard() {
  return AdaptiveCardBuilder.create()
    .withVersion('1.6')
    .addInputChoiceSet((i) =>
      i
        .withId('people-picker')
        .withLabel('Select users in the whole organization')
        .isMultiSelect()
        .withValue('user1,user2')
        .withChoicesData('graph.microsoft.com/users'),
    )
    .addAction((a) => a.submit('Submit'))
    .build();
}
